src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -3153,6 +3155,40 @@ paths:
         '404':
           description: |
             **nodeId** does not exist
+  '/nodes/{nodeId}/storage-info/{contentPropName}':
+    get:
+      x-alfresco-since: "7.2.0"
+      tags:
+        - storage-info
+      summary: Retrieve storage properties for given content
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.2.0 and newer versions.
+
+        Gets storage properties for given content.
```

Comment:
Maybe mention that this only applies if a compatible version of the cloud storage connector is installed (eg. "Alfresco Content Connector for AWS S3"). Otherwise, returns empty response.

Could / should we give some example responses ?

Reply:
Added.